WordPress/gutenberg
# List: Disable edit as HTML support

## What?

This PR disables HTML editing support for list blocks.
A similar PRs: #11785, #39318 and #49426.

## Why?

> Editing these wrapper blocks in HTML directly is fragile and prone to breaking the users' content. HTML editing should be left to leaf-blocks.

https://github.com/WordPress/gutenberg/pull/11785#issue-380011111

## Testing Instructions





1. Open a Post or Page.
2. Insert a List block.
3. Confirm "Edit as HTML" action isn't available in block Options.

## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/42362903/0ef42377-3e6c-43b8-958e-d0cf54ff9650
